src/ansys_sphinx_theme/announcement_layout.html

```diff
@@ -0,0 +1,14 @@
+{% if theme_switcher %} 
```

Comment:
This logic is working, @Revathyvenugopal162. If no file is found, it will just print a message in the console but will not break the style.

I will update the version mapper to recursively add the `announcement.html` inside each folder of a release that is not the static one. This should do the job.

Could you please update the CSS file to apply for the `id="announce"`?

See how it is rendered now: https://templates.ansys.com/release/0.1/index.html